public/app/screens/main/sections/devices/device/controls/slider/slider.jsx

```diff
@@ -13,16 +12,23 @@ var SliderControl = React.createClass({
     };
   },
 
-  render: function () {
+  render: function render () {
     return (
       <div className="cc-slider-control">
-        <input className="cc-slider-control__input" type="range" min={this.state.min} max={this.state.max} step="1" value={this.state.value} onChange={this.onChange} />
+        <input
+          className="cc-slider-control__input"
+          type="range"
+          min={this.state.min}
+          max={this.state.max}
+          step="1"
+          value={this.state.value}
+          onChange={this.onChange} />
       </div>
```

Comment:
I'm not OK with breaking every single attribute into single lines. This blows up length of code. Programmers should decide if they want to wrap lines or not. There shouldn't be "wrap always" rule.